GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -33,6 +32,11 @@ public override void ConfigureVisualStudio(string gitBinPath, ITracer tracer)
         public override bool TryGetGVFSHooksPathAndVersion(out string hooksPaths, out string hooksVersion, out string error)
         {
             hooksPaths = string.Empty;
+            string binPath = $"/usr/local/vfsforgit/{GVFSPlatform.Instance.Constants.GVFSHooksExecutableName}";
+            if (File.Exists(binPath))
+            {
+                hooksPaths = binPath;
+            }
 
             // TODO(POSIX): Get the hooks version rather than the GVFS version (and share that code with the Windows platform)

```

Comment:
A few questions:

- On Windows we find the location of the hooks using `where`.  What was the reason for hardcoding it here (rather than using `which`)?
- Can this `TODO` be addressed now?  Previously we were blocked by not having the hooks path.  Now that we do, can the code in the Windows platform for reading the version of the hooks be shared?

Reply:
1. On the Mac `which` is not able to find `GVFS.Hooks.exe` today - 
```
$ which gvfs.hooks
-bash: type: gvfs.hooks: not found
```

This is because it does not exist in the standard search paths. We can discuss the installer options to enable `which` (may be install a link like we do already for `gvfs`, or update `$PATH`). This is not directly related to this PR (service porting and auto-mount on Mac). I have created an [issue](https://github.com/Microsoft/VFSForGit/issues/1028) to track this. We can do this in a separate PR.

2. This is not related to service porting and auto-mount. Have created a tracking [issue](https://github.com/Microsoft/VFSForGit/issues/1044).